### DISCUSSION TITLE
Install and make vim-hexokinase.

### DISCUSSION THREAD

**Discussion by santigo-dev** [wbthomason/packer.nvim#838] [Q&A] [answered]
Hello, I'm having some trouble installing [this plugin](https://github.com/RRethy/vim-hexokinase), because it requires you to run `make hexokinase` at the root of the plugin after you install it, I can do this manually and the plugin works, but I don't know how to do this last part of running make with packer, can someone please help me out?

I tried doing this but it doesn't seem to work:
```lua
-- Vim Hexokinase
use {
    'RRethy/vim-hexokinase',
    run = 'make hexokinase',
}
```
And lastly I should add that I'm on nightly.

**Comment by sdondley** [comment, marked as answer]
I would try a `cd` into the directory the file is located in and then try the command:

`run = cd /path/to/.config/nvim/site/pack/packer/start/vim-hexokinase && make hexokinase`

Change the 'startt' directory to 'opt' in the path above if necessary (depends if you have `opt` set to `true`).

**Reply by santigo-dev** [reply]
It seems to be working, thanks!

**Reply by AidanJHMurphy** [reply]
What does this phrase mean [run is treated as a shell command and run in the installation directory of the plugin via $SHELL -c '\<run\>'](https://github.com/wbthomason/packer.nvim#updateinstall-hooks)

It sounds like that would mean that the run command should already be executed in the installation directory of the plugin that it is associated with, which would make that cd redundant. That apparent described behavior would make more sense to me, so that nobody's run command would be fragile with respect to changes that Packer might make to it's installation/implementation.

Are you sure that there isn't just something wrong with the way packer handles this plugin or the make command? I'm asking because I'm trying to do this same thing, and the cd really smells fishy to me.
